torchci/lib/bot/cliParser.ts

```diff
@@ -56,6 +56,10 @@ mergeOption.add_argument("-l", "--land-checks", {
   "reliability and decrease risk of revert. The tests added are: pull, Lint and trunk. Note " +
   "that periodic is excluded. (EXPERIMENTAL)",
 });
+mergeOption.add_argument("-r", "--rebase", {
```

Comment:
Should we accept branch name here, i.e. `-r viable/strict` to make this consistent in term of functionality with `@pytorchbot rebase -s|-b BRANCHNAME`.  The default branch could be set to `master`.

Reply:
done!